api/onnx_web/chain/tile.py

```diff
@@ -125,24 +128,43 @@ def blend_tiles(
         scaled_left = left * scale
 
         # equalized size may be wrong/too much
-        scaled_bottom = min(scaled_top + equalized.shape[0], scaled_size[0])
-        scaled_right = min(scaled_left + equalized.shape[1], scaled_size[1])
-        logger.trace(
+        scaled_bottom = scaled_top + equalized.shape[0]
+        scaled_right = scaled_left + equalized.shape[1]
+        
+        writable_top = max(scaled_top,0)
+        writable_left = max(scaled_left,0)
+        writable_bottom = min(scaled_bottom,scaled_size[0])
+        writable_right = min(scaled_right,scaled_size[1])
+        
+        margin_top = writable_top - scaled_top
+        margin_left = writable_left - scaled_left
+        margin_bottom = writable_bottom - scaled_bottom
+        margin_right = writable_right - scaled_right
+        
+        logger.debug(
```

Comment:
Having two back-to-back log messages at the same log level seems odd to me. Can these be combined into one message, or should one of them be at a different log level?

Reply:
They can be combined, I was just being lazy.

Reply:
Fixing